client-sdk/go/types/transaction.go

```diff
@@ -419,9 +442,77 @@ type PrettyTransaction struct {
 	AuthInfo AuthInfo   `json:"ai"`
 }
 
+func (ptx *PrettyTransaction) PrettyPrint(ctx context.Context, prefix string, w io.Writer) {
+	fmt.Fprintf(w, "%sFormat: %s\n", prefix, ptx.Call.Format)
+
+	if ptx.Call.Method != "" {
+		fmt.Fprintf(w, "%sMethod: %s\n", prefix, ptx.Call.Method)
+	}
+
+	fmt.Fprintf(w, "%sBody:\n", prefix)
+	// If the body type supports pretty printing, use that.
+	if pp, ok := ptx.Call.Body.(prettyprint.PrettyPrinter); ok {
+		pp.PrettyPrint(ctx, prefix+"  ", w)
+	} else {
+		// Otherwise, just serialize into JSON and display that.
+		data, err := json.MarshalIndent(ptx.Call.Body, prefix+"  ", "  ")
+		if err != nil {
+			fmt.Fprintf(w, "%s<error: %s>\n", prefix+"  ", err)
+		}
+		fmt.Fprintf(w, "%s%s\n", prefix+"  ", data)
+	}
+
+	for _, si := range ptx.AuthInfo.SignerInfo {
```

Comment:
This seems a bit strange, just outputting a bunch of nonces? Shouldn't we have a section called "Signers" or similar and then have entries for every signer (also showing other data)?